.rubocop.yml

```diff
@@ -13,9 +14,12 @@ AllCops:
 # Metrics/LineLength:
 #   Max: 120
 
+
 Style/StringLiterals:
   Enabled: true
   EnforcedStyle: single_quotes
+  Exclude:
+    - 'Gemfile'  # until engine_cart stanza is violation free
```

Comment:
Is this just as a reminder to update the Gemfile later? It seems like we could make the Gemfile modifications w/o causing any violations (perhaps just quote issues?)